docs/source/tutorials/arch_modeling/open_cell_libraries_tutorial.rst

```diff
@@ -0,0 +1,480 @@
+Build an FPGA fabric using Standard Cell Libraries
+==================================================
+
+Introduction
+~~~~~~~~~~~~
+
+**In this tutorial, we will**
+   - Showcase how to create an architecture description based on standard cells, using OpenFPGA's circuit modeling language
+   - Use Skywater's Process Development Kit (`PDK`_) cell library to create an OR Gate circuit model for OpenFPGA

```

Comment:
It is Process Design Kit rather than Process Development Kit. https://en.wikipedia.org/wiki/Process_design_kit#:~:text=A%20process%20design%20kit%20(PDK,technology%20variation%20for%20their%20processes.